ldms/src/store/darshan/Plugin_darshan_stream_store.man

```diff
@@ -0,0 +1,81 @@
+.\" Manpage for Plugin_darshan_stream_store
+.\" Contact ovis-help@ca.sandia.gov to correct errors or typos.
+.TH main 7 "26 September 2021" "v4" "LDMS Plugin darshan_stream_store man page"
+
+.SH NAME
+Plugin_darshan_stream_store - man page for the LDMS darshan_stream_store plugin
+
+.SH SYNOPSIS
+Within ldmsd_controller or a configuration file:
+.br
+config name=darshan_stream_store [ <attr>=<value> ] 
+
+.SH DESCRIPTION
+With LDMS (Lightweight Distributed Metric Service), plugins for the ldmsd (ldms daemon) are configured via ldmsd_controller or a configuration file. 
+The darshan_stream_store plugin writes out either a single darshan JSON stream's data to the SOS container. The input data produced by the LDMS darshan plugin and consists of two types of messages "MOD" for module data and "MET for metadata. Both messages are saved into the same SOS container.
```

Comment:
plugin and --> plugin. The "and" connector is unnecessary. Add a ':' after the word "messages".